src/main/java/frc/robot/commands/climber/ZeroClimber.java

```diff
@@ -0,0 +1,32 @@
+package frc.robot.commands.climber;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.Climber;
+
+public class ZeroClimber extends Command{
+    private Climber climber;
+    public ZeroClimber(Climber climber){
+        this.climber = climber;
+    }
+    
+    @Override
+    public void initialize(){
+        climber.zeroed = false;
```

Comment:
This is why you don't want to set an internal variable to a value outside the class.

Shouldn't the climber initialize it's state to zeroed? Like in the configClimber method?